scheduler/pmc-exporter-task/src/main/kotlin/uk/ac/ebi/scheculer/pmc/exporter/model/Link.kt

```diff
@@ -0,0 +1,19 @@
+package uk.ac.ebi.scheculer.pmc.exporter.model
+
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlProperty
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlRootElement
+
+private const val PROVIDER_ID = 1518
+
+data class Link(
```

Comment:
small details but given all this model class are so small (and simple) have them all in single file with faciliate their understading.